docker-compose/initdb/init.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<-EOSQL
-    CREATE USER "aggregate" WITH UNENCRYPTED PASSWORD 'aggregate';
```

Comment:
Good catch. There is a similar line in the README. Do we also want that there?

Reply:
I'll throw another commit in with that too :)